README.md

```diff
@@ -15,11 +15,20 @@ Logs are output in log directory where you can tail them. To see all logs when s
 If something goes wrong kafka/zookeeper may have to be manually closed. To do this just run `./close.sh`
 If this doesn't work you can try manually killing the process: `for pid in $(ps aux | grep kafka | grep brevity | awk '{print $2}'); do kill -9 $pid; done`
 
+## Setup Heroku Local and running
+1. Install Heroku CLI (source: https://devcenter.heroku.com/articles/heroku-cli)
+2. Login to Heroku using the credentials
```

Comment:
what credentials? do we make our own accounts or do we need to use a specific account to get any premium features?

Reply:
yeah will give it in chat, forgot to mention

Reply:
let's test this out first before merging, I updated the README